scripts/install/wireguard.sh

```diff
@@ -150,6 +150,8 @@ AllowedIPs = 0.0.0.0/0
 #PersistentKeepalive = 25
 EOWGC
 
+    sudo chown -R "$u": /home/"$u"/.wireguard
```

Comment:
There is no need for `sudo` here. The script runs as root.

Reply:
Sniiky sniiky sudo
```suggestion
    chown -R "$u": /home/"$u"/.wireguard
```